### PR TITLE
[FR] Update `MWC/2017_4K`

### PR DESCRIPTION
**Pull request by Soetch**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by DoryanR** [comment]
Hi, for me there is not only the tags which are to be added, several incoherence this is between the French and English version, it misses whole sections.

If I should give you an advice, I would tell you to take the english file, copy it entirely, and replace the content of the french file by the one you just copied, and you re-translate.

This will be the easiest way for tournament articles.

- [ ] Please do not merge until the article is really updated.

**Comment by Soetch** [comment]
> Hi, for me there is not only the tags which are to be added, several incoherence this is between the French and English version, it misses whole sections.
> 
> If I should give you an advice, I would tell you to take the english file, copy it entirely, and replace the content of the french file by the one you just copied, and you re-translate.
> 
> This will be the easiest way for tournament articles.
> 
>     * [ ]  Please do not merge until the article is really updated.

Gosh, I'm working on it then.

**Comment by Soetch** [comment]
> Hi, for me there is not only the tags which are to be added, several incoherence this is between the French and English version, it misses whole sections.
> 
> If I should give you an advice, I would tell you to take the english file, copy it entirely, and replace the content of the french file by the one you just copied, and you re-translate.
> 
> This will be the easiest way for tournament articles.
> 
>     * [ ]  Please do not merge until the article is really updated.

The article is updated, the "groups" section and podium was missing. Sorry for making you losing time like this.

**Comment by TPGPL** [comment]
updated the translation to be up-to-date information-wise, should be ready to review now

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
 
-Dans chaque coupe du monde, les joueurs qui réussissent à obtenir une place sur le podium reçoivent des prix uniques créés spécialement pour les gagnants de la coupe du monde. Il est possible que ces récompenses changent entre plusieurs coupe du monde.
-
 | Place | Récompenses |
 | :-- | :-- |
-| ![Gold Crown](/wiki/shared/crown-gold.png "1re place") | 150 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil, et le titre « osu!mania Champion » pendant un an |
-| ![Silver Crown](/wiki/shared/crown-silver.png "2e place") | 80 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |
-| ![Bronze Crown](/wiki/shared/crown-bronze.png "3e place") | 40 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |
+| ![Couronne en or](/wiki/shared/crown-gold.png "1ère place") | 150 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil, et le titre « osu!mania Champion » pendant un an |
+| ![Couronne en argent](/wiki/shared/crown-silver.png "2ème place") | 80 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:33]:
```suggestion
| ![Couronne d'argent](/wiki/shared/crown-silver.png "2ème place") | 80 $ par membre de l'équipe, un badge de profil unique, de la marchandise exclusive d'osu! |
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
+  - MWC 2017
+  - MWC 4K 2017
+  - MWC2017
+  - MWC4K 2017
 ---
 
 # osu!mania 4K World Cup 2017
 
-![osu!mania 4K World Cup 2017](img/logo.jpg)
+![MWC 4K 2017 Logo](img/logo.jpg)
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:11]:
```suggestion
![Logo de la MWC 4K 2017](img/logo.jpg)
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 3]
```diff
@@ -1,39 +1,43 @@
...
 
 La **osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania entre pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de la osu!mania World Cup.
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:13]:
```suggestion
L'**osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania 4 par pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de l'osu!mania 4K World Cup.
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 4]
```diff
@@ -1,39 +1,43 @@
...
 ## Récompenses
 
-Dans chaque coupe du monde, les joueurs qui réussissent à obtenir une place sur le podium reçoivent des prix uniques créés spécialement pour les gagnants de la coupe du monde. Il est possible que ces récompenses changent entre plusieurs coupe du monde.
-
 | Place | Récompenses |
 | :-- | :-- |
-| ![Gold Crown](/wiki/shared/crown-gold.png "1re place") | 150 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil, et le titre « osu!mania Champion » pendant un an |
-| ![Silver Crown](/wiki/shared/crown-silver.png "2e place") | 80 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |
-| ![Bronze Crown](/wiki/shared/crown-bronze.png "3e place") | 40 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |
+| ![Couronne en or](/wiki/shared/crown-gold.png "1ère place") | 150 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil, et le titre « osu!mania Champion » pendant un an |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:32]:
```suggestion
| ![Couronne en or](/wiki/shared/crown-gold.png "1ère place") | 150 $ par membre de l'équipe, un badge de profil unique, de la marchandise exclusive d'osu!, le titre "osu!mania Champion" pendant un an |
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 5]
```diff
@@ -1,39 +1,43 @@
...
+| ![Couronne en argent](/wiki/shared/crown-silver.png "2ème place") | 80 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |
+| ![Couronne en bronze](/wiki/shared/crown-bronze.png "3ème place") | 40 $ par membre de l'équipe, pins et objets osu! exclusifs, un badge sur le profil |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:34]:
```suggestion
| ![Couronne de bronze](/wiki/shared/crown-bronze.png "3ème place") | 40 $ par membre de l'équipe, un badge de profil unique, de la marchandise exclusive d'osu! |
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 6]
```diff
@@ -1,39 +1,43 @@
...
+
+![](img/badge.jpg "MWC 4K 2017 winner badge") ![](img/badge2.jpg "MWC 4K 2017 2nd place badge") ![](img/badge3.jpg "MWC 4K 2017 3rd place badge")
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:36]:
```suggestion
![](img/badge.jpg "Badge des gagnants de la MWC 4K 2017") ![](img/badge2.jpg "Badge de la 2ème place de la MWC 4K 2017") ![](img/badge3.jpg "Badge de la 3ème place de la MWC 4K 2017")
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 7]
```diff
@@ -1,39 +1,43 @@
...
 
 ## Organisation
 
-La coupe du monde osu!mania 2017 est organisée par divers membres de la communauté, la multitude de tâches est distribuée dans plusieurs domaines de responsabilité.
+L'osu!mania 4K World Cup 2017 était organisée par plusieurs membres de la communauté
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:40]:
```suggestion
L'osu!mania 4K World Cup 2017 était organisée par divers membres de la communauté.
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 8]
```diff
@@ -46,45 +50,10 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
 
 - [Topic de discussion](https://osu.ppy.sh/community/forums/topics/615787)
 - [Diffusion en direct](https://www.twitch.tv/osulive)
-- [Supportez votre équipe avec une bannière de profil !](https://osu.ppy.sh/store/products/226)
 - **[Statistiques générales, groupes et plus!](https://docs.google.com/spreadsheets/d/1qAbKpW3yCJ5U2qUoUoIob6qsmNW08LenxeL0FptdjxA/pubhtml)**
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:53]:
```suggestion
- **[Statistiques générales, groupes et plus !](https://docs.google.com/spreadsheets/d/1qAbKpW3yCJ5U2qUoUoIob6qsmNW08LenxeL0FptdjxA/pubhtml)**
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 9]
```diff
@@ -46,45 +50,10 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
 
 - [Topic de discussion](https://osu.ppy.sh/community/forums/topics/615787)
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:51]:
```suggestion
- [Fil de discussion](https://osu.ppy.sh/community/forums/topics/615787)
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 10]
```diff
@@ -120,10 +89,52 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| **A** | ::{ flag=BR }:: Brésil | ::{ flag=PL }:: Polognee | ::{ flag=PH }:: Philippines | ::{ flag=VE }:: Venezuela |
+| **B** | ::{ flag=GB }:: Royaume-Uni | ::{ flag=MY }:: Malaisie | ::{ flag=HK }:: Hong Kong | ::{ flag=NO }:: Norvège |
+| **C** | ::{ flag=CN }:: Chine | ::{ flag=FR }:: France | ::{ flag=AU }:: Australie | ::{ flag=TW }:: Taiwan |
+| **D** | ::{ flag=US }:: États-Unis | ::{ flag=AR }:: Argentine | ::{ flag=MX }:: Mexique | ::{ flag=DE }:: Allemagne |
+| **E** | ::{ flag=KR }:: Corée du Sud | ::{ flag=SE }:: Suède | ::{ flag=RU }:: Russian Federation | ::{ flag=FI }:: Finland |
+| **F** | ::{ flag=ID }:: Indonesie | ::{ flag=IT }:: Italie | ::{ flag=ES }:: Espagne | ::{ flag=NZ }:: Nouvelle-Zélande |
+| **G** | ::{ flag=JP }:: Japon | ::{ flag=CA }:: Canada | ::{ flag=CH }:: Suisse | ::{ flag=NL }:: Pays-Bas |
+| **H** | ::{ flag=CL }:: Chili | ::{ flag=TH }:: Thaïlande | ::{ flag=SG }:: Singapour | ::{ flag=BE }:: Belgique |
+
+![](img/bracket.jpg "MWC 4K 2017 bracket")
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:105]:
```suggestion
![](img/bracket.jpg "Bracket de la MWC 4K 2017")
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 11]
```diff
@@ -120,10 +89,52 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+
+## Podium
+
+![](img/podium.jpg "MWC 4K 2017 podium")
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:109]:
```suggestion
![](img/podium.jpg "Podium de la MWC 4K 2017")
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 12]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+
+Samedi, 16 Septembre 2017:
+
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
+| --: | :-: | :-: | :-- | :-- |
+| États-Unis ::{ flag=US }:: | 0 | **7** | ::{ flag=KR }:: **Corée du Sud** | [#1](https://osu.ppy.sh/community/matches/36111799) |
+
+### Finales - Semaine 1
+
+Samedi, 9 Septembre 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:234]:
```suggestion
Samedi 9 septembre 2017 :
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 13]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
+| --: | :-: | :-: | :-- | :-- |
+| **Japon** ::{ flag=JP }:: | **6** | 0 | ::{ flag=CL }:: Chili | [#1](https://osu.ppy.sh/community/matches/35952218) |
+| Royaume-Uni ::{ flag=GB }:: | 0 | **6** | ::{ flag=BR }:: **Brésil** | [#1](https://osu.ppy.sh/community/matches/35955973) |
+
+Dimanche, 10 Septembre 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:241]:
```suggestion
Dimanche 10 septembre 2017 :
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 14]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
+| --: | :-: | :-: | :-- | :-- |
+| **Corée du Sud** ::{ flag=KR }:: | **6** | 0 | ::{ flag=US }:: États-Unis | [#1](https://osu.ppy.sh/community/matches/35972975) |
+| **Brésil** ::{ flag=BR }:: | **6** | 1 | ::{ flag=JP }:: Japon | [#1](https://osu.ppy.sh/community/matches/35974418) |
+| Brésil ::{ flag=BR }:: | 5 | **6** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35998794) |
+
+### Demi-finale
+
+Samedi, 2 Septembre 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:251]:
```suggestion
Samedi 2 septembre 2017 :
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 15]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
+| --: | :-: | :-: | :-- | :-- |
+| Suède ::{ flag=SE }:: | 1 | **6** | ::{ flag=JP }:: **Japon** | [#1](https://osu.ppy.sh/community/matches/35769761) |
+| **Chine** ::{ flag=CN }:: | **6** | 2 | ::{ flag=FR }:: France | [#1](https://osu.ppy.sh/community/matches/35771726) |
+| Brésil ::{ flag=BR }:: | 0 | **6** | ::{ flag=KR }:: **Corée du Sud** | [#1](https://osu.ppy.sh/community/matches/35774130) |
+| Pologne ::{ flag=PL }:: | 1 | **6** | ::{ flag=ID }:: **Indonésie** | [#1](https://osu.ppy.sh/community/matches/35777392) |
+| **Royaume-Uni** ::{ flag=GB }:: | **6** | 3 | ::{ flag=CA }:: Canada | [#1](https://osu.ppy.sh/community/matches/35779902) |
+| **États-Unis** ::{ flag=US }:: | **6** | 0 | ::{ flag=CL }:: Chili | [#1](https://osu.ppy.sh/community/matches/35783065) |
+
+Dimanche, 3 Septembre 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:262]:
```suggestion
Dimanche 3 septembre 2017 :
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 16]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
+| --: | :-: | :-: | :-- | :-- |
+| **Japon** ::{ flag=JP }:: | **6** | 3 | ::{ flag=ID }:: Indonésie | [#1](https://osu.ppy.sh/community/matches/35804308) |
+| Chine ::{ flag=CN }:: | 2 | **6** | ::{ flag=GB }:: **Royaume-Uni** | [#1](https://osu.ppy.sh/community/matches/35807134) |
+
 ### Quarts de finale
 
-| Samedi 26 août 2017 |  |  |  |  |
+Samedi, 26 Août 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:271]:
```suggestion
Samedi 26 août 2017 :
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 17]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| Équipe 1 |  |  | Équipe 2 | Lien du match |
 | --: | :-: | :-: | :-- | :-: |
-| France ::{ flag=FR }:: | 4 | 5 | ::{ flag=CL }:: Chili | [#1](https://osu.ppy.sh/community/matches/35585664) |
-| Argentine ::{ flag=AR }:: | 1 | 5 | ::{ flag=GB }:: Royaume-Uni | [#1](https://osu.ppy.sh/community/matches/35588367) |
-| Canada ::{ flag=CA }:: | 0 | 5 | ::{ flag=US }:: États-Unis | [#1](https://osu.ppy.sh/community/matches/35590366) |
+| France ::{ flag=FR }:: | 4 | **5** | ::{ flag=CL }:: **Chili** | [#1](https://osu.ppy.sh/community/matches/35585664) |
+| Argentine ::{ flag=AR }:: | 1 | **5** | ::{ flag=GB }:: **Royaume-Uni** | [#1](https://osu.ppy.sh/community/matches/35588367) |
+| Canada ::{ flag=CA }:: | 0 | **5** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35590366) |
+
+Dimanche, 27 Août 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:279]:
```suggestion
Dimanche 27 août 2017 :
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 18]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| **Corée du Sud** ::{ flag=KR }:: | **5** | 0 | ::{ flag=JP }:: Japon | [#1](https://osu.ppy.sh/community/matches/35601021) |
+| Nouvelle-Zélande ::{ flag=NZ }:: | 0 | **5** | ::{ flag=PL }:: **Pologne** | [#1](https://osu.ppy.sh/community/matches/35602514) |
+| Singapour ::{ flag=SG }:: | 1 | **5** | ::{ flag=CN }:: **Chine** | [#1](https://osu.ppy.sh/community/matches/35604433) |
+| Hong-Kong ::{ flag=HK }:: | 0 | **5** | ::{ flag=SE }:: **Suède** | *victoire par défaut* |
+| **Brésil** ::{ flag=BR }:: | **5** | 1 | ::{ flag=ID }:: Indonésie | [#1](https://osu.ppy.sh/community/matches/35608732) |
 
 ### Huitièmes de finale
 
-| Dimanche 20 août 2017 |  |  |  |  |
+Dimanche, 20 Août 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:291]:
```suggestion
Dimanche 20 août 2017 :
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 19]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| **Corée du Sud** ::{ flag=KR }:: | **5** | 0 | ::{ flag=AR }:: Argentine | [#1](https://osu.ppy.sh/community/matches/35421542) |
+| **Brésil** ::{ flag=BR }:: | **5** | 0 | ::{ flag=SG }:: Singapour | [#1](https://osu.ppy.sh/community/matches/35423100) |
+| Hong-Kong ::{ flag=HK }:: | 4 | **5** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/35424487) |
+| Pologne ::{ flag=PL }:: | 0 | **5** | ::{ flag=CL }:: **Chili** | [#1](https://osu.ppy.sh/community/matches/35429710) |
+| Suède ::{ flag=SE }:: | 0 | **5** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35431890) |
 
 ### Phases de poules
 
-| Samedi 12 août 2017 |  |  |  |  |
+Samedi, 12 Août 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:306]:
```suggestion
Samedi 12 août 2017 :
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 20]
```diff
@@ -210,90 +221,145 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| Pologne ::{ flag=PL }:: | 0 | **4** | ::{ flag=BR }:: **Brésil** | [#1](https://osu.ppy.sh/community/matches/35218496) |
+| Allemagne ::{ flag=DE }:: | 0 | **4** | ::{ flag=MX }:: **Mexique** | [#1](https://osu.ppy.sh/community/matches/35219133) |
+| Argentine ::{ flag=AR }:: | 1 | **4** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35218503) |
+| Pays-Bas ::{ flag=NL }:: | 1 | **4** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/35218508) |
+| Venezuela ::{ flag=VE }:: | 0 | **4** | ::{ flag=PL }:: **Pologne** | [#1](https://osu.ppy.sh/community/matches/35219912) |
+| Allemagne ::{ flag=DE }:: | 0 | **4** | ::{ flag=US }:: **États-Unis** | [#1](https://osu.ppy.sh/community/matches/35219931) |
+| Mexique ::{ flag=MX }:: | 0 | **4** | ::{ flag=AR }:: **Argentine** | [#1](https://osu.ppy.sh/community/matches/35219934) |
+| Suisse ::{ flag=CH }:: | 0 | **4** | ::{ flag=CA }:: **Canada** | [#1](https://osu.ppy.sh/community/matches/35219940) |
+
+Dimanche, 13 Août 2017:
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:338]:
```suggestion
Dimanche 13 août 2017 :
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 21]
```diff
@@ -1,41 +1,45 @@
...
+  - MWC4K 2017
 ---
 
 # osu!mania 4K World Cup 2017
 
-![osu!mania 4K World Cup 2017](img/logo.jpg)
+![Logo de la MWC 4K 2017](img/logo.jpg)
 
-La **osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania entre pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de la osu!mania World Cup.
+L'**osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania 4 par pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de l'osu!mania 4K World Cup.
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:13]:
```suggestion
L'**osu!mania 4K World Cup 2017** (***MWC 4K 2017***) est un tournoi d'osu!mania 4K par pays organisé par [l'équipe osu!](/wiki/People/The_Team). Il s'agit de la quatrième édition de l'osu!mania 4K World Cup.
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 22]
```diff
@@ -120,10 +89,52 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
 | ::{ flag=US }:: | **États-Unis** | **[Zyph](https://osu.ppy.sh/users/1600432)**, [EtienneXC](https://osu.ppy.sh/users/5610085), [TheToaphster](https://osu.ppy.sh/users/7616811), [Chrubble](https://osu.ppy.sh/users/2594280), [stupud man](https://osu.ppy.sh/users/2141612), [xElectroGH](https://osu.ppy.sh/users/3251373) |
 | ::{ flag=VE }:: | **Venezuela** | **[Cyssel](https://osu.ppy.sh/users/6394478)**, [Satoruu](https://osu.ppy.sh/users/2243452), [Shinsekai-](https://osu.ppy.sh/users/2140739), [Marunne](https://osu.ppy.sh/users/8841450), [Prancer](https://osu.ppy.sh/users/1821350), [\[-Gearfrik-\]](https://osu.ppy.sh/users/8198818) |
 
-![Double Elimination bracket](img/bracket.jpg)
+## Groupes
+
+| Groupe | Top seed | High seed | Low seed | Unseeded |
+| :-: | :-- | :-- | :-- | :-- |
+| **A** | ::{ flag=BR }:: Brésil | ::{ flag=PL }:: Polognee | ::{ flag=PH }:: Philippines | ::{ flag=VE }:: Venezuela |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:96]:
```suggestion
| **A** | ::{ flag=BR }:: Brésil | ::{ flag=PL }:: Pologne | ::{ flag=PH }:: Philippines | ::{ flag=VE }:: Venezuela |
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 23]
```diff
@@ -120,10 +89,52 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| **B** | ::{ flag=GB }:: Royaume-Uni | ::{ flag=MY }:: Malaisie | ::{ flag=HK }:: Hong Kong | ::{ flag=NO }:: Norvège |
+| **C** | ::{ flag=CN }:: Chine | ::{ flag=FR }:: France | ::{ flag=AU }:: Australie | ::{ flag=TW }:: Taiwan |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:98]:
```suggestion
| **C** | ::{ flag=CN }:: Chine | ::{ flag=FR }:: France | ::{ flag=AU }:: Australie | ::{ flag=TW }:: Taïwan |
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 24]
```diff
@@ -120,10 +89,52 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| **D** | ::{ flag=US }:: États-Unis | ::{ flag=AR }:: Argentine | ::{ flag=MX }:: Mexique | ::{ flag=DE }:: Allemagne |
+| **E** | ::{ flag=KR }:: Corée du Sud | ::{ flag=SE }:: Suède | ::{ flag=RU }:: Russian Federation | ::{ flag=FI }:: Finland |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:100]:
```suggestion
| **E** | ::{ flag=KR }:: Corée du Sud | ::{ flag=SE }:: Suède | ::{ flag=RU }:: Fédération de Russie | ::{ flag=FI }:: Finlande |
```

File: wiki/Tournaments/MWC/2017_4K/fr.md [hunk 25]
```diff
@@ -120,10 +89,52 @@ La coupe du monde osu!mania 2017 est organisée par divers membres de la communa
...
+| **F** | ::{ flag=ID }:: Indonesie | ::{ flag=IT }:: Italie | ::{ flag=ES }:: Espagne | ::{ flag=NZ }:: Nouvelle-Zélande |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/MWC/2017_4K/fr.md:101]:
```suggestion
| **F** | ::{ flag=ID }:: Indonésie | ::{ flag=IT }:: Italie | ::{ flag=ES }:: Espagne | ::{ flag=NZ }:: Nouvelle-Zélande |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**